src/cv2pdb.vcxproj

```diff
@@ -208,8 +208,18 @@
       <ExcludedFromBuild Condition="'$(Configuration)|$(Platform)'=='Release|Win32'">true</ExcludedFromBuild>
     </MASM>
   </ItemGroup>
+  <ItemGroup>
+    <None Include="packages.config" />
+  </ItemGroup>
   <Import Project="$(VCTargetsPath)\Microsoft.Cpp.targets" />
   <ImportGroup Label="ExtensionTargets">
     <Import Project="$(VCTargetsPath)\BuildCustomizations\masm.targets" />
+    <Import Project="packages\Microsoft.VisualStudio.Setup.Configuration.Native.1.16.30\build\native\Microsoft.VisualStudio.Setup.Configuration.Native.targets" Condition="Exists('packages\Microsoft.VisualStudio.Setup.Configuration.Native.1.16.30\build\native\Microsoft.VisualStudio.Setup.Configuration.Native.targets')" />

```

Comment:
I think it would be much easier to just include Setup.Configuration.h into the src folder (the license seems to allow this), so building is independent from nuget packages.
Some people also prefer to compile cv2pdb with tools other than VS, e.g. https://github.com/rainers/cv2pdb/pull/37

Reply:
It is still nice the header can automatically be retrieved from the source. What about including the retrieved package in the repository? Those who would like to know the source or update one can still check `package.config`. Those who don't have NuGet just doesn't have to be bothered with it.